### PR TITLE
feat: Modified home page and changed app icon

### PR DESCRIPTION
**Pull request by Priyanka-Sachan**
ANWESHA_SCREENSHOT
Modified home page and changed app icon(issue #8 and#9).

### CODE REVIEW

**Review by s-ayush2903 — APPROVED** [review]

**Review by s-ayush2903 — APPROVED** [review]

**Comment by atm1504** [comment]
@Priyanka-Sachan I don't see any code change that can bring the UI changes that you have attached.

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -6,258 +6,175 @@
...
-                android:text="Sponsors"
-                android:textColor="@android:color/white"
-                android:textSize="16sp" />
-
-        </LinearLayout>
+        android:layout_height="match_parent"
+        android:scaleType="centerCrop"
+        android:src="@drawable/back" />
+    <ImageView
+        android:layout_width="400dp"
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_home.xml]: Remove all hardcoded dimensions. Define the dimensions in the dimens.xml file and use them here.

**Comment by Priyanka-Sachan** [comment]
Anwesha_Screenshot
fixes issue #8 and #9

**Comment by Priyanka-Sachan** [comment]
Anwesha_Home
fixes issue #8 and #9.
Image slider would be added .

**Review by atm1504 — APPROVED** [review]

**Pull request merged**